react/Badge/Badge.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import classnames from 'classnames';
+
+import styles from './Badge.less';
+
+type Tone =
+  | 'accent'
+  | 'critical'
+  | 'info'
+  | 'neutral'
+  | 'positive'
+  | 'secondary';
+
+type Props = {
+  children: React$Node,
+  tone?: Tone,
+  primary?: boolean
+};
+
+export default class Badge extends PureComponent<Props> {
+  static displayName = 'Badge';
+
+  props: Props;
+
+  render() {
+    const { children, tone, primary } = this.props;
+
+    return (
+      <div
+        className={classnames({
+          [styles.root]: true,
+          [styles[tone]]: tone,
+          [styles.primary]: primary
```

Comment:
The previous implementation deliberately stayed away from using `primary`/`secondary` as the relationship is not necessarily hierarchical, rather more to do with visual weight. @stephaniemoss?

Reply:
Yeah, this is a problem. We need to get together to discuss what the logic is behind the visual weight as it applies to things like buttons and alerts.

Reply:
How about `strong`?